### PR TITLE
Spec branch1

### PR DESCRIPTION
**Pull request by ankitthakur01**
changed blog template. Added 
blog folder, post1, post2, post3, post4 html files and changes to blog.css

### CODE REVIEW

File: blog/fonts/fontawesome-webfont.svg [hunk 1]
```diff
@@ -0,0 +1,685 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (sjainit) [inline blog/fonts/fontawesome-webfont.svg]: @ankitthakur01, same for this. Try to cut the unnecessary lines if possible.

File: blog/css/style.css [hunk 2]
```diff
@@ -0,0 +1,3669 @@
+/*----------------------------------------------------
```
[the remark this answers is not in the document]
Comment (hiteshsandal) [reply blog/css/style.css]: @ankitthakur01 delete all the comments from this file.

File: blog/css/style.css [hunk 3]
```diff
@@ -0,0 +1,3669 @@
...
+  margin-top: 25px;
+}
+
+.mt-100 {
+  margin-top: 100px;
+}
+
+.p0 {
+  padding-left: 0px;
+  padding-right: 0px;
```
Comment (hiteshsandal) [inline blog/css/style.css]: @ankitthakur01 look for such type of styling which is of no use for our page.

**Comment by hiteshsandal** [comment]
@ankitthakur01 remove navbar from all the subpages which you named them as postx.html. Instead of the navbar, give one nav-link back to the blog page from each subpage.

File: post1.html [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+<!-- w3-content defines a container for fixed size centered content,
+and is wrapped around the whole page content, except for the footer in this example -->
+<div class="w3-content" style="max-width:1400px">
+
+  <!-- Header -->
+  <header class="w3-container w3-center w3-padding-32">
+    <h1><b>SPEC BLOG</b></h1>
+    <p style="letter-spacing:1px;">Welcome to the blog of <span class="w3-tag">Electronics</span></p>
+  </header>
+
```
Comment (hiteshsandal) [inline post1.html]: No need to add this to any of the subpages. To get design ideas for subpages, look for the current WordPress page of blogs.

File: blog.html [hunk 5]
```diff
@@ -1,291 +1,302 @@
...
+                        <div class="media post_item">
+                            <img src="blog/img/popular-post/post4.jpg" alt="post">
+                            <div class="media-body">
+                                <a href="post4.html"><h3>Best Coding Challenges Websites</h3></a>
+                            </div>
+                        </div>
+                        <div class="br"></div>
+                    </aside>
+                </div>
+            </div>
```
Comment (hiteshsandal) [inline blog.html]: @ankitthakur01 The size of this section is not as per the page. Style font of the page where you are styling body in blog.css

**Pull request closed without merging**